src/types.jl

```diff
@@ -77,6 +77,8 @@ function exprtype(x::BasicSymbolic)
     end
 end
 
+wvd = WeakValueDict{UInt, BasicSymbolic}()
```

Comment:
Is there a benefit to not making this `const`? Wouldn't it lead to non-const global issues?

Reply:
Updated. Thank you!